src/Camera.ts

```diff
@@ -18,34 +18,38 @@ export class Camera {
     'DEATH_FAIL_IN_EFFECT_SHAKE',
   ];
 
-  private handle: number;
+  private _handle: number;
```

Comment:
Not sure why you changed this to _handle, can you change it back?

Reply:
Because that is naming convention that I am used to and follow and have followed for this repository. Convention is that class properties, methods that are private should have underscore prefix.

Reply:
The only place this kind of syntax is used is in Vehicle*.ts, the main reason I don't want to use this kind of syntax is I'm used underscores being used to suppress unused propertys

Reply:
Ah, nvm, I see. Guess properties were changed for this repository and I did not notice. Changed it back 👍